instances/differenceMeasure/meanSquaredError.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+    "@context": {
+      "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/differenceMeasure/meanSquaredError",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/DifferenceMeasure",
+    "definition": "The mean squared difference between two series of values.",
+    "description": null,
+    "interlexIdentifier": null,
+    "knowledgeSpaceLink": null,
+    "name": "z-score",
```

Comment:
copy paste error: should be "mean squared error"
does this conflict / duplicate the TypeOfUncertainty term ?

Reply:
fixed copy-paste error